project.clj

```diff
@@ -5,7 +5,7 @@
     :password :env/clojars_jenkins_password
     :sign-releases false })
 
-(defproject puppetlabs/certificate-authority "0.6.1-SNAPSHOT"
+(defproject puppetlabs/certificate-authority "0.6.2-SNAPSHOT"
```

Comment:
We have a release job on Jenkins that does this step automatically and uses the old version number as the version for the release. So this change should be rolled back.